packages/flutter/lib/src/gestures/events.dart

```diff
@@ -410,8 +414,9 @@ class PointerHoverEvent extends PointerEvent {
     Offset delta = Offset.zero,
     int buttons = 0,
     bool obscured = false,
-    double pressureMin = 1.0,
-    double pressureMax = 1.0,
+    double pressure = 0.0,
+    double pressureMin = 0.0,
+    double pressureMax = 6.0,
```

Comment:
this seems wrong too. By definition a hover event can't have any pressure. Surely all three numbers should be 1.0 (like they were before).